server/lib/chat/chat-service.ts

```diff
@@ -515,10 +516,7 @@ export default class ChatService {
     for (const c of channels.values()) {
       this.publisher.publish(getChannelPath(c), {
         action: 'userOffline',
-        user: {
-          id: userId,
-          name: user.name,
-        },
+        userId,
```

Comment:
You're changing the structure of this event, so you need to pass it on a separate path (`action` in this case) or you're going to break existing clients

Reply:
Apply throughout this file, there's a bunch of these cases